cmd/ako-main/main.go

```diff
@@ -295,23 +250,26 @@ func InitializeAKC() {
 
 	akoControlConfig.SetLicenseType(aviRestClientPool.AviClient[0])
 
+	if lib.GetAdvancedL4() {
+		err, seGroupToUse := lib.FetchSEGroupWithMarkerSet(aviRestClientPool.AviClient[0])
+		if err != nil {
+			utils.AviLog.Warnf("Setting SEGroup with markerset failed: %s", err)
+		}
+		if seGroupToUse == "" {
+			utils.AviLog.Infof("Continuing with Default-Group SEGroup")
+			seGroupToUse = lib.DEFAULT_SE_GROUP
+		}
+		lib.SetSEGName(seGroupToUse)
+	}
+
 	err = c.HandleConfigMap(informers, ctrlCh, stopCh, quickSyncCh)
 	if err != nil {
 		utils.AviLog.Errorf("Handle configmap error during reboot, shutting down AKO. Error is: %v", err)
 		return
 	}
 
-	if !utils.IsVCFCluster() {
-		if _, err := lib.GetVipNetworkListEnv(); err != nil {
-			utils.AviLog.Fatalf("Error in getting VIP network %s, shutting down AKO", err)
-		}
-	} else {
-		lib.NewVCFDynamicClientSet(cfg)
-		lslrMap, _ := lib.GetNetinfoCRData()
-		for _, lr := range lslrMap {
-			lib.SetT1LRPath(lr)
-			break
-		}
+	if _, err := lib.GetVipNetworkListEnv(); !utils.IsVCFCluster() && err != nil {
```

Comment:
can we have just if condition of previous code? as this code will rub `vipNetworkList` call even in vcf cluster?